src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Model/KnownTypeSymbols.cs

```diff
@@ -13,6 +13,8 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
 {
     internal sealed record KnownTypeSymbols
     {
+        public CSharpCompilation Compilation { get; }
```

Comment:
`CSharpCompilation` is not an equatable type so this breaks `record`.

Reply:
Not meant to be incremental; will switch to `class`.